packages/flutter/lib/src/services/text_input.dart

```diff
@@ -447,6 +447,7 @@ class TextInputConfiguration {
   /// [actionLabel] may be null.
   const TextInputConfiguration({
     this.inputType = TextInputType.text,
+    this.readOnly = false,
```

Comment:
As we discussed yesterday offline, we can add an assert to line 462 with a message "readonly fields cannot autofill".

Reply:
Done.